### ISSUE TITLE
Text refreshing when hovering translation suggestions

### ISSUE THREAD

**Issue by ItielMaN** [vslavik/poedit#261]
Hello,

Poedit seems to refresh the text when hovering the translation suggestions.
STR:
1. Take any string that has a translation suggestion(s).
2. Hover back and forth from a translation suggestion text to the other texts in the same suggestion (percent, Ctrl + X, machine translation/human translation strings)
3. Looks like the text is being refreshed.
I tried to capture it in a gif but it's not demonstrating the issue very well (except for once when hovering the third suggestion's "תרגום מכונה" (machine translation) of the string with the "Finder" text in it. But it should be 100% reproducible):
![animation](https://cloud.githubusercontent.com/assets/18016656/14154593/eca0e872-f6c5-11e5-906d-8d981991c83a.gif)

Basically it's the same issue as here, just for hovering and not clicking:
http://forums.mydigitallife.info/threads/64027-Spartan-Toolkit-Still-in-development?p=1205710&viewfull=1#post1205710
(See comments 107, 109, 111, 112, 114, 118 & 119)

Yeah I know it shouldn't be considered as a bug but I would call it an annoyance :)

Thanks.

**Comment by vslavik** [comment]
It’s unclear to me whether you’re talking about the *text* itself, or the tooltip — it sounds like the former, but only the latter is visible in the GIF. I don’t think I can do anything about the latter and I believe the former is addressed by ea6538c7d73889983236d813104ed21496983ed5 (post-1.8.7).

**Comment by ItielMaN** [comment]
> It’s unclear to me whether you’re talking about the text itself, or the tooltip

The text itself. The tooltip's behaviour is OK.

> I believe the former is addressed by ea6538c (post-1.8.7).

Umm, not sure about the "to avoid flicker when resizing windows or splitters" part as that's exactly what I'm seeing:
![animation](https://cloud.githubusercontent.com/assets/18016656/14187953/36b193d0-f78e-11e5-8c1c-8e7d29567579.gif)
(It actually looks much better in real time. This gif has less frames I guess, which causes it to look worse)

I tried to make a new gif but still it's hard to see the issue with it. But technically it looks the same as the image above.

Thanks.

**Comment by vslavik** [comment]
Should be fixed in this build: https://jumpshare.com/v/CuhDhrLDAQvYLigNG2sz

**Comment by ItielMaN** [comment]
Yes, looks OK now :)
Though I'm not sure if the issue you've mentioned in your first post here is fixed. Flickering still appears, but I don't think it matters.
